src/Avalonia.Visuals/Media/FormattedText.cs

```diff
@@ -159,13 +159,10 @@ public IEnumerable<Rect> HitTestTextRange(int index, int length)
         }
 
         /// <summary>
-        /// Gets the size of the text, taking <see cref="Constraint"/> into account.
+        /// Gets the bounds of the text, taking <see cref="Constraint"/> into account.
         /// </summary>
-        /// <returns>The bounds box of the text.</returns>
-        public Size Measure()
-        {
-            return PlatformImpl.Size;
-        }
+        /// <returns>The bounds of the text.</returns>
+        public Rect Measure() => PlatformImpl.Bounds;
```

Comment:
We will never calculate anything here. Text layouts are immutable. The moment we construct one measures are fixed. I suggest changing this to a property `public Rect Bounds { get; }` or just use `PlatformImpl.Bounds`

Reply:
I was going to say "we got that API from WPF", but it seems we didn't! Yes, will change it to a `Bounds` property.

Reply:
Done!